nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/SplitText.java

```diff
@@ -94,12 +118,16 @@
             .addValidator(StandardValidators.NON_NEGATIVE_INTEGER_VALIDATOR)
             .defaultValue("0")
             .build();
+    public static final PropertyDescriptor HEADER_MARKER = new PropertyDescriptor.Builder()
+            .name("Header Line Marker Characters")
+            .description("The first character(s) on the line of the datafile which signifies a header line. This value is ignored when Header Line Count is non-zero. " +
+                    "The first line not containing the Header Line Marker Characters and all subsequent lines are considered non-header")
+            .required(false)
+            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
+            .build();
     public static final PropertyDescriptor REMOVE_TRAILING_NEWLINES = new PropertyDescriptor.Builder()
             .name("Remove Trailing Newlines")
-            .description("Whether to remove newlines at the end of each split file. This should be false if you intend to merge the split files later. If this is set to "
-                + "'true' and a FlowFile is generated that contains only 'empty lines' (i.e., consists only of \r and \n characters), the FlowFile will not be emitted. "
-                + "Note, however, that if the Header Line Count is greater than 0, the resultant FlowFile will never be empty as it will consist of the header lines, so "
-                + "a FlowFile may be emitted that contians only the header lines.")
+            .description("Whether to remove newlines at the end of each split file. This should be false if you intend to merge the split files later.")
```

Comment:
@markobean I think the longer description is still valid and should remain in place.